lib/pdk/cli/util.rb

```diff
@@ -150,6 +159,7 @@ def puppet_from_opts_or_env(opts)
         gemset.each do |gem, version|
           next if version.nil?
 
+          next if logging_disabled
```

Comment:
I'd put this conditional outside of the loop as there's no need to enter the loop at all if you're not logging.

Reply:
Updated in latest push.